assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -268,10 +271,14 @@ def run(args):
         log_time_and_message("Reading data in spark and converting to pandas")
         baseline_df = try_read_mltable_in_spark_with_error(args.baseline_data, "baseline_data")
 
+        raw_categorical_features = get_categorical_cols_with_df_with_override(baseline_df,
```

Comment:
this variable isn't used anywhere else. if this is meant to be used once the comment on line 281 is addressed, can we comment this line out and add a TODO here too. Otherwise, it might unnecessarily introduce issues and is confusing

Reply:
updated the change